src/itest/kotlin/nl/info/zac/itest/config/ItestConfiguration.kt

```diff
@@ -416,6 +416,12 @@ object ItestConfiguration {
      */
     lateinit var zaakProductaanvraag1Uuid: UUID
 
+    /**
+     * Global variable to store the UUID of a zaaktype that is created in the integration tests
+     * from a productaanvraag.
+     */
+    lateinit var zaakTypeProductaanvraag1UuId: UUID
```

Comment:
This variable never gets set in the integration tests yet... You need to set it first before you can read it in your itests. That is why the itests fail. Let me try to fix that for you @DaxRiool :-)

Reply:
actually this entire variable should be removed. zaaktype UUIDs are static things and we already have a constant for it.

Reply:
fixed this now.